scripts/unit_test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Copyright 2020 The ElasticDL Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -e
+
+(
+    cd /tmp/elasticdl
+    go test -v -cover ./...
+)
+
+pytest elasticdl/python/tests elasticdl_preprocessing/tests \
```

Comment:
I think this script cannot execute alone.
How about name it build_and_test.sh and call build.sh in this script.

Reply:
Good Suggestion!